lms/views/api/blackboard/sync.py

```diff
@@ -0,0 +1,17 @@
+from pyramid.view import view_config
+
+from lms.security import Permissions
+
+
+class Sync:
+    def __init__(self, request):
+        self._request = request
+
+    @view_config(
+        route_name="blackboard_api.sync",
+        request_method="POST",
+        renderer="json",
+        permission=Permissions.API,
+    )  # pylint: disable=no-self-use
+    def sync(self):
```

Comment:
I'm adding here a useless view to allow for `req.route_path("blackboard_api.sync")` 

I think adding all the boilerplate for it here it's not a bad idea.